pkg/apis/alerting/v1alpha/alerting.proto

```diff
@@ -0,0 +1,206 @@
+syntax = "proto3";
+option go_package = "github.com/rancher/opni/pkg/alerting/v1alpha";
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/struct.proto";
+import "google/protobuf/empty.proto";
+import "google/protobuf/descriptor.proto";
+import "google/api/http.proto";
+import "google/api/annotations.proto";
+import "google/rpc/status.proto";
+import "github.com/rancher/opni/pkg/apis/core/v1/core.proto";
+
+service Alerting {
+  // ------- Trigger Alerts -------
+  // TODO
+  rpc TriggerAlerts(TriggerAlertsRequest) returns (TriggerAlertsResponse) {
+    option (google.api.http) = {
+      post: "/alerts/v1alpha/trigger"
```

Comment:
Don't include the version in the path, if we add multiple versions it will be handled in router middleware